Python/graph.py

```diff
@@ -245,7 +245,7 @@ def diffuse_p1(p1, start_node, G, visited_nodes, threshold_diff, adj_mat, verbos
     start_node_unvisited_nbors = [node for node in start_node_nbors if node not in visited_nodes]
 
     if start_node_unvisited_nbors:
-        w_edges_sum = sum(adj_mat2.loc[start_node_unvisited_nbors, start_node])
+        w_edges_sum = sum(abs(adj_mat2.loc[start_node_unvisited_nbors, start_node]))
```

Comment:
Edges_sum should be the sum of absolute values so we can use it for normalizing edge weights.